packages/components/src/styles/components/application-state.scss

```
@@ -7,43 +7,82 @@
 // APPLICATION-STATE COMPONENT
 //
 
-$hds-application-state-padding: 12px 0;
+$hds-application-state-content-max-width: 480px;
 
 .hds-application-state {
-  width: 19.5rem;
-  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  width: fit-content;
   margin: 0 auto; // this will center the component in the parent container
+
+  &.hds-application-state--align-center {
+    align-items: center;
+    text-align: center;
+
+    .hds-application-state__header,
+    .hds-application-state__body,
+    .hds-application-state__footer {
+      width: auto;
+    }
+  }
+}
+
+.hds-application-state__media {
+  margin-bottom: 32px;
+
+  > * {
+    display: block;
+    max-width: 100%;
+  }
 }
 
 .hds-application-state__header {
   display: grid;
+  grid-template-rows: auto auto;
   grid-template-columns: min-content 1fr;
```

Not a blocker,  but we tend to use `grid-template-areas` because it's much easier to reason about:
```suggestion
  grid-template-areas:
    "error error"
    "icon title";
```